as/itests/org.jboss.tools.as.ui.bot.itests/src/org/jboss/tools/as/ui/bot/itests/parametized/server/ShowInBrowserTest.java

```diff
@@ -0,0 +1,38 @@
+package org.jboss.tools.as.ui.bot.itests.parametized.server;
+
+import static org.junit.Assert.assertTrue;
+import org.eclipse.reddeer.eclipse.wst.server.ui.cnf.Server;
+import org.eclipse.reddeer.eclipse.wst.server.ui.cnf.ServersView2;
+import org.eclipse.reddeer.junit.runner.RedDeerSuite;
+import org.eclipse.reddeer.requirements.server.ServerRequirementState;
+import org.eclipse.reddeer.swt.api.Browser;
+import org.eclipse.reddeer.swt.api.Menu;
+import org.eclipse.reddeer.swt.impl.browser.InternalBrowser;
+import org.eclipse.reddeer.swt.impl.menu.ContextMenu;
+import org.jboss.ide.eclipse.as.reddeer.server.requirement.ServerRequirement.JBossServer;
+import org.jboss.tools.as.ui.bot.itests.AbstractTest;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(RedDeerSuite.class)
+@JBossServer(state=ServerRequirementState.RUNNING)
+public class ShowInBrowserTest extends AbstractTest {
+	
+	@Test
+	public void checkBrowserText() throws InterruptedException {
```

Comment:
I would maybe even go a bit further and break down one (e2e) test case into a couple of test cases.
Test case 1: Stopped server has context menu item "Show In" available, but "Web Browser" sub menu item is "disabled". So this could be a negative check.
Test case 2: Server is started. Context menu item Show In and Web Browser is available and is "clickable". 
Test case 3: Server is started. Invoking Show In -> Web Browser opens Browser and it contains proper text.

If you will think about it, it will need some more code, some Before/After methods implementation, changing state of the server in the requirement.